experimental/framework/data-objects/src/signalManager/README.md

```diff
@@ -0,0 +1,91 @@
+
+
+### Creation
+Just like with DDSes, the `FluidContainer` provides a container schema where you can include `SignalManager` as a shared object you would like to load.
+
+Here is a look at how you would go about loading `SignalManager` as part of the initial objects of the container:
+
+```typescript
+const containerSchema: ContainerSchema = {
+    initialObjects: {
+        signalManager: SignalManager,
+    },
+};
+
+const { container, services } =  await client.createContainer(containerSchema);
+
+const signalManager = container.initialObjects.signalManager as SignalManager
+```
+
+`signalManager` can then be directly used in your Fluid application!
+
+If you are loading an existing container the process is the same except you would use `getContainer` instead of `createContainer`:
+
+```typescript
+const containerSchema: ContainerSchema = {
+    initialObjects: {
+        signalManager: SignalManager,
+    },
+};
+
+const { container, services } =  await client.getContainer(id, containerSchema);
+
+const signalManager = container.initialObjects.signalManager as SignalManager
+```
+
+
+To dynamically create `SignalManager` instances it is the same process as with DDSes as well. You add the `SignalManager` type to `dynamicObjectTypes` in the container schema and then use the `create` function on the container:
+
+```typescript
+const containerSchema: ContainerSchema = {
+    /*...*/
+    dynamicObjectTypes: [ SignalManager ]
+};
+
+const { container, services } =  await client.getContainer(id, containerSchema);
+
+const newSignalManager = await container.create(SignalManager) //Creates a new SignalManager instance
+```
+
+
+
+### API
+`SignalManager` provides a few simple methods to send signals and add/remove listeners to specific signals as well:
+- `submitSignal(signalName: string, payload?: Jsonable)` - Sends a signal with a payload to its connected listeners
+- `onSignal(signalName: string, listener: SignalListener)` - Adds a listener for the specifies signal. Same behavior as EventEmitter's `on` method.
+- `offSignal(signalName: string, listener: SignalListener | ((message: any) => void))` - Removes a listener for the specified signal. Same behavior as EventEmitter's `off` method.
+
+
+
+
+### Common Patterns
+#### Signal Request
+There are some scenarios where a client would like to request a specific signal be sent to them from other connected clients within the application. One main use case for this pattern is when a newly joining client needs to recieve pertinent information immediatley after connecting the container. For example, in the [FocusTracker](https://github.com/microsoft/FluidFramework/tree/main/examples/data-objects/focus-tracker) we define a new focus request signal type that is used when a newly joining client requests to recieve the focus-state of each currently connected client:
```

Comment:
Leading with the need and following with the approach can make it easier to follow sometimes.

```suggestion
When a client joins a collaboration session, they may need to receive pertinent information immediately after connecting the container.  To support this, they can request a specific signal be sent to them from other connected clients within the application. For example, in the [FocusTracker](https://github.com/microsoft/FluidFramework/tree/main/examples/data-objects/focus-tracker) we define a "focusRequest" signal type that a newly joining client uses to request the focus-state of each currently connected client:
```